hawc/main/urls.py

```diff
@@ -74,6 +74,11 @@
         views.AdminAssessmentSize.as_view(),
         name="admin_assessment_size",
     ),
+    url(
```

Comment:
let's chat about this a bit; I was thinking of just making the standard vanilla admin views which require very little to no programming effort